drivers/npmx/npmx_shell.c

```diff
@@ -1792,6 +1798,176 @@ static int cmd_adc_meas_take_vbat(const struct shell *shell, size_t argc, char *
 	return 0;
 }
 
+static int pof_config_get(const struct shell *shell, size_t argc, char **argv,
+			  pof_config_type_t config_type)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
+
+	if (npmx_instance == NULL) {
+		shell_error(shell, "Error: shell is not initialized.");
+		return 0;
+	}
+
+	npmx_pof_t *pof_instance = npmx_pof_get(npmx_instance, 0);
+
+	npmx_pof_config_t pof_config;
+	npmx_error_t err_code = npmx_pof_config_get(pof_instance, &pof_config);
+
+	if (check_error_code(shell, err_code)) {
+		switch (config_type) {
+		case POF_CONFIG_TYPE_ENABLE:
+			shell_print(shell, "Value: %d.", (int)pof_config.status);
+			break;
+		case POF_CONFIG_TYPE_POLARITY:
+			shell_print(shell, "Value: %d.", (int)pof_config.polarity);
+			break;
+		case POF_CONFIG_TYPE_THRESHOLD:
+			uint32_t mVolts;
+			if (npmx_pof_threshold_convert_to_mv(pof_config.threshold, &mVolts)) {
+				shell_print(shell, "Value: %d mV.", mVolts);
+			} else {
+				shell_error(shell,
+					    "Error: unable to convert threshold value to mV.");
```

Comment:
```suggestion
					    "Error: unable to convert threshold value to millivolts.");
```

Reply:
I prefer to stay with the current notation. This is consistent with the SI system, and also with the rest of the messages in this file.

Reply:
@annakielar can you help? Based on what I found, `mV` can be used with value, but when we talk about the unit, we should use a full name `millivolt`.

Reply:
There are 6 other places in this file where we use this naming convention. Does it make sense to change in this one place to another version?
If so, maybe I'll put the changes in a new commit, so as not to post changes not related to this PR here?

Reply:
As Witek suggested, when there's no number mentioned, we should use full forms. If that's not a big deal, please correct it at least in this file - otherwise, in the nearest future I'll correct outstanding occurrences in all nrfx and npmx files.